packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -222,6 +233,30 @@ export class SCWSigner implements Signer {
     }
   }
 
+  private async getCachedWalletConnectResponse(): Promise<WalletConnectResponse | null> {
```

Comment:
Wonder if we should start moving things away from the SCWSigner into smaller helper functions.